applications/launchpad_v2/src/containers/BaseNodeContainer/index.tsx

```diff
@@ -20,6 +29,26 @@ const BaseNodeContainer = () => {
   return (
     <div>
       <h2>Base Node</h2>
+      <div style={{padding: '16px'}}>
+        <Select
+          value={tariNetwork}
+          options={networkOptions}
+          onChange={setTariNetwork}
+          label="Tari network"
+        />
+      </div>
+
+
```

Comment:
Here are 2 empty lines - another inconsistency between prettier and eslint. Maybe we should add `no-multiple-empty-lines` rule?

Reply:
I connected prettier to eslint and ran lint:fix, it should be good now